### PR TITLE
Update CONTRIBUTING.md file

### PR DESCRIPTION
**Pull request by pradeep288**
Closes: #2708

## Description

Please include a summary of the change and which issue is fixed. Please also
include relevant motivation and context. List any dependencies that are required
for this change.

Closes: #(issue-number)

## Type of change

Please mark options that are relevant:

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to
  not work as expected)
- [ ] This change requires a documentation update
- [ ] Build related change

## How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide
instructions so we can reproduce. If applicable, please also list any relevant
details for your test configuration.

Not Applicable, this is markdown file change
## Checklist:

- [x] My code follows the CONTRIBUTION [guidelines](./../CONTRIBUTING.md) of
  this project
- [ ] I have commented on my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -58,6 +58,8 @@ $ git add <files>
 $ git commit -m "govc: Add feature ..." -m "Closes: #<issue-number>"
 $ git push $USER issue-<number>
 ```
+**Note**:  
+To register the new `govc` command package, import package into `govmomi/govc/main.go` file.
```
Comment (embano1) [inline CONTRIBUTING.md:62]:
```suggestion
To register the new `govc` command package, add a blank `_` import to `govmomi/govc/main.go`.
```

**Review by embano1 — CHANGES_REQUESTED** [review]
Minor change

**Review by embano1 — COMMENTED** [review]
THX! Please squash and I'll approve.

**Comment by embano1** [comment]
Something seems wrong with the commit.

Please also note that:

- There's a typo in the commit title
- Please use the `docs: ` prefix in the commit title for proper CHANGELOG rendering.

**Comment by pradeep288** [comment]
> Something seems wrong with the commit.
> 
> Please also note that:
> 
> * There's a typo in the commit title
> * Please use the `docs: ` prefix in the commit title for proper CHANGELOG rendering.

Updated the commit message as per the above suggestion

**Review by embano1 — APPROVED** [review]
LGTM

**Pull request merged**